### PR TITLE
Persist

### PR DESCRIPTION
**Pull request by GMSoudersJr**
😨 This one is big. My bad. I don't know why the test results are also included.
Some tests were refactored to make their locators more descriptive and less flaky.
As far as the login, at the moment, the login form only saves the username to the local storage. In the future, it will ping the database to check usernames for uniqueness.

https://github.com/GMSoudersJr/card-workout/assets/44438849/5072f62f-bb56-47ec-bbd4-eabcd3d2945c

Persisting a workout consists of saving the workout after the last card has been discarded. When the start button is pressed, the timer start is set and when the last card is discarded, the timer end is set. That compromises a workout. Eventually there should be a name added to the workout

persist workout and time

### CODE REVIEW

File: Session.vim
Comment (drecali) [inline Session.vim:1]: @GMSoudersJr is this file necessary? If not, I'd add it to your `.gitignore`

Comment (GMSoudersJr) [reply Session.vim:1]: I have been debating about this file for a few days. I need to get better at making it before the commit, so that it matches the last commit. NeoVim makes a copy of the buffers I am in so that I can open up to the same place (files, window sizes, buffers, etc...) when I open NeoVim again. It is really useful for when I work from different computers.

Comment (drecali) [reply Session.vim:1]: That's impressive! I can definitely see the utility, especially for a terminal wizard such as yourself

**Comment by GMSoudersJr** [comment]
Added cards to display workout name (which will be added when exercises are grouped into decks for faster workout selection), and a link to the view previous workouts.

workoutCard

The workout card allows for two actions:
1. Repeat button for repeating the same workout. Navigates to the cards page and is ready to go.
2. Delete the workout. Asks for confirmation, updates the local storage, transitions the deletion, navigates to home page if no workouts are left.

homepageWithActivitiesLink
The Link is only visible when there are saved workouts in local storage.

**Comment by GMSoudersJr** [comment]
Whoa, those pics are HUGE!!
